### PR TITLE
Add globbing support for config files for mugc

### PR DESCRIPTION
**Pull request by ArtemK007**
Hey, would be great, if we can specify shell glob pattern for config files, or specify list of paths and extension of files to construct list of policies. I've made dirty implementation here (I'm not a developer), do you think it good enough to merge? :)

### CODE REVIEW

**Comment by ArtemK007** [comment]
I've tried to rework this with positional args, but looks like it's not so easy. For example you can't easily provide multiple positional args to match policies in different locations, for example we have our policies in subdirs, like:
- `example/a/*.yml`
- `example/b/*.yml`
- `example/c/*.yml`

so, with current implementation I could say:
`mugc.py -c example/* -e yml` and it will get all policies in `a/b/c` subdirs.

At the same time use can specify single file and glob pattern with extension matching like that:
`mugc.py -c another_dir/policy.yml -c example/* -e yml`
and this will also work fine.

So I think my solution is still better, than one with positional args. What do you think?

**Comment by kapilt** [comment]
also would you mind signing the project cla, its referenced/linked from the bottom of the project readme, direct link https://docs.google.com/forms/d/19LpBBjykHPox18vrZvBbZUcK6gQTj7qv1O5hCduAZFU/viewform

Thanks

**Comment by kapilt** [comment]
re positional expansion, a shell syntax for that expansion is `example/*/*yml`

File: tools/ops/mugc.py [hunk 1]
```diff
@@ -104,10 +105,18 @@ def resources_gc_prefix(options, policy_collection):
 
 
 def setup_parser():
+    class ExtendAction(argparse.Action):
```
Comment (kapilt) [inline tools/ops/mugc.py]: could you clarify the need for extend over append in this context?

**Comment by kapilt** [comment]
so given custodian `run` and `validate` also use multivalue positional args for config files, it seems better to focus on that usage mode, and also provides for common unix cli patterns instead of doing something idiosyncratic.

**Comment by ArtemK007** [comment]
Thanks a lot!

**Pull request merged**